### ISSUE TITLE
Peak deconvolution 

### ISSUE THREAD

**Issue by jphysycs** [barahona-research-group/RamanSPy#3]
Hi, I've some questions about peak deconvolution using Ramanspy, in order to fit Raman data.

Is there a function to do this type of analysis ?
If yes, could you give some examples pls

Thanks for the attention.

**Comment by dgeorgiev21** [comment]
Thank you for reaching out!

`RamanSPy` provides methods for hyperspectral unmixing (sometimes referred to as deconvolution), but not for peak deconvolution at this time. 

However, the package has been built with extensibility in mind, allowing users to integrate new methods. 

Do you have any particular methods you would like to see in `RamanSPy`? 

---
If you find `RamanSPy` useful, please consider starring the project on `GitHub`! Your support means a lot!

**Comment by jphysycs** [comment]
Yes, something like that: https://github.com/MyPyDavid/raman-fitting/blob/main/examples/fitting_Model_1st_6peaks%2BSi.png

It is possible in Ramanspy, find the peak then fit Gaussian or Lorentzian ?

Thanks for the attention.
